backend/controllers/productController.js

```
@@ -0,0 +1,187 @@
+      filter.subCategory = { $in: subCategory.split(",") };
+    }
+
+    let sortOption = {};
+    if (sort === "lowHigh") sortOption.price = 1;
+    if (sort === "highLow") sortOption.price = -1;
+
+    const products = await productModel
+      .find(filter)
+      .sort(sortOption)
+      .skip(skip)
+      .limit(parseInt(limit));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)